tests/elements-regression/tests/templates/button.json

```diff
@@ -1 +1 @@
aphy_font_family":"Roboto","typography_font_weight":"500","text_shadow_text_shadow_type":"yes","text_shadow_text_shadow":{"horizontal":13,"vertical":-14,"blur":4,"color":"rgba(0,0,0,0.3)"},"button_text_color":"#060606","background_color":"#F5F7F5"},"elements":[],"isInner":false,"widgetType":"button","elType":"widget"},{"id":"4333e389","settings":{"text":"Icon, spacing 20, position after","selected_icon":{"value":"far fa-arrow-alt-circle-right","library":"fa-regular"},"icon_align":"right","icon_indent":{"unit":"px","size":20,"sizes":[]},"typography_typography":"custom","typography_font_family":"Roboto","typography_font_weight":"500","text_padding":{"unit":"px","top":"10","right":"10","bottom":"10","left":"10","isLinked":true}},"elements":[],"isInner":false,"widgetType":"button","elType":"widget"},{"id":"41af2471","settings":{"text":"Text G fonts - sec, g colour - primary, border Type - solid, width all 10, border radius all 10","border_border":"solid","border_width":{"unit":"px","top":"10","right":"10","bottom":"10","left":"10","isLinked":true},"border_radius":{"unit":"px","top":"10","right":"10","bottom":"10","left":"10","isLinked":true},"__globals__":{"typography_typography":"","button_text_color":"globals\/colors?id=secondary","background_color":"globals\/colors?id=primary"},"typography_typography":"custom","typography_font_family":"Roboto","typography_font_weight":"600","button_text_color":"#54595F","background_color":"#6EC1E4","button_box_shadow_box_shadow_type":"yes","button_box_shadow_box_shadow":{"horizontal":13,"vertical":-15,"blur":14,"spread":4,"color":"rgba(0,0,0,0.5)"},"align":"right"},"elements":[],"isInner":false,"widgetType":"button","elType":"widget"},{"id":"11da4ea3","settings":{"text":"Text color custom #030303, color custom - #E1D626, border type -groove, padding all 10","button_text_color":"#030303","background_color":"#E1D626","border_border":"groove","text_padding":{"unit":"px","top":"10","right":"10","bottom":"10","left":"10","isLinked":true},"typography_typography":"custom","typography_font_family":"Roboto","typography_font_weight":"500"},"elements":[],"isInner":false,"widgetType":"button","elType":"widget"},{"id":"518be7b0","settings":{"text":"Gradient location 40, radial, position top left, boarder type double, width all 10, colour cusom #010102","background_background":"gradient","background_color_stop":{"unit":"%","size":40,"sizes":[]},"background_gradient_type":"radial","background_gradient_position":"top left","border_border":"double","border_width":{"unit":"px","top":"10","right":"10","bottom":"10","left":"10","isLinked":true},"border_color":"#010102","typography_typography":"custom","typography_font_family":"Roboto","typography_font_weight":"500","text_padding":{"unit":"px","top":"13","right":"13","bottom":"13","left":"13","isLinked":true}},"elements":[],"isInner":false,"widgetType":"button","elType":"widget"}],"isInner":false,"elType":"container"}],"page_settings":[],"version":"0.4","title":"button","type":"page"}
```

Comment:
@rami-elementor My request would be to create a separate PR in which you update the test, and to merge that PR first.

In this way we can see better that we are not breaking something.

I am hesitant to make these changes at the same time.

Wdyt?

Reply:
@hein-obox Regression test moved to a separate PR: https://github.com/elementor/elementor/pull/28812